tests/xml_parser_test.c

```diff
@@ -95,6 +95,25 @@ static int s_xml_parser_child_with_text_test(struct aws_allocator *allocator, vo
 
 AWS_TEST_CASE(xml_parser_child_with_text, s_xml_parser_child_with_text_test)
 
+const char *malformed = "<?xml version=\"1.0\" encoding=\"UTF-8\"?><rootNode>><child1>TestBody</child1></rootNode>";
+
+static int s_xml_parser_malformed_test(struct aws_allocator *allocator, void *ctx) {
```

Comment:
nit: i would call out exactly what is malformed in this test, in case we want to add more malformed tests

Reply:
Thanks, updated.